apps/guide/content/docs/legacy/popular-topics/display-components.mdx

```diff
@@ -32,6 +32,10 @@ Text Display components let you add markdown-formatted text to your message and
 	Sending user and role mentions in text display components **will notify users and roles**! You can and should control
 	mentions with the `allowedMentions` message option.
 </Callout>
+<Callout>
+	Text display components can be used in [modals](../interactions/modals#text-display). If you wanna learn how to, see
+	the modal guide.
```

Comment:
- avoid super informal speech like "wanna"
- link to the relevant page when referring to other sections